sparse/layout_test.go

```diff
@@ -1,192 +0,0 @@
-package sparse
```

Comment:
We will need test cases for similar functions anyway, to make sure the file layout is correct after sync.